kafdrop.tf

```diff
@@ -15,11 +15,15 @@ resource "cloudfoundry_app" "kafdrop" {
   space        = data.cloudfoundry_space.space.id
   memory       = 1024
   disk_quota   = 2048
-  docker_image = "obsidiandynamics/kafdrop"
+  docker_image = "skprasad/kafdrop"
```

Comment:
We can move this to `philipslabs` repo and have this build using Github Actions. What is the source?

Reply:
https://github.com/krishna-prasad-s/kafdrop,  I even have an Travis CI setup.. need to update credentials

Reply:
Forked to https://github.com/philips-labs/kafdrop -- I've attached the Docker hub credentials as well